src/test/migrations/steps/20230404102924-step-recalculate-other-land.ts

```diff
@@ -0,0 +1,65 @@
+import * as pgPromise from 'pg-promise'
+
+import { AreaController } from '@server/controller/area'
+import { AssessmentController } from '@server/controller/assessment'
+import { BaseProtocol, Schemas } from '@server/db'
+
+import { getRows } from '@test/migrations/steps/utils/getRow'
+
+import { calculateRow } from './utils/calculateRow'
+
+export default async (client: BaseProtocol) => {
+  const { assessment, cycle } = await AssessmentController.getOneWithCycle(
+    {
+      assessmentName: 'fra',
+      metaCache: true,
+      cycleName: '2025',
+    },
+    client
+  )
+
+  const variableName = 'otherLand'
+  const tableName = 'extentOfForest'
+
+  const rows = await getRows({ tableName, variableName, cycle, assessment }, client)
```

Comment:
if you pass one variable, shouldn't the method getRows return only one row?

Reply:
yas, it should.

Reply:
done